basket/news/tasks.py

```diff
@@ -248,6 +248,7 @@ def fxa_verified(data):
 
     locale = data.get('locale')
     subscribe = data.get('marketingOptIn')
+    newsletters = data.get('newsletters')
```

Comment:
Just making sure `newsletters` doesn't need a default value. If not, this looks good to go to me!

Reply:
the default value returned from `get()` is `None`, and below I'm just doing a check to see if it's truthy, which would fail if it were none or an empty list (which is what FxA will send if people select no newsletters). So I believe we're good, and there are tests of this method without the `newsletters` key.